medicaltorch/datasets.py

```diff
@@ -187,23 +219,29 @@ def get_pair_slice(self, slice_index, slice_axis=2):
             "gt_metadata": gt_meta_dict,
         }
 
+        if self.metadata:
```

Comment:
Not sure to understand that section, could you help us @andreanne-lemay ?

Reply:
I integrated this ivadomed method : https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/loader.py#L31. The `idx` is because now there is a list of metadata for the multichannel option.